### ISSUE TITLE
Display twinkling after reboot - HD 4600 + GT 755M

### ISSUE THREAD

**Issue by hyperbroly** [Floris497/mac-pixel-clock-patch-V2#212]
Hi,

Thank you for your patch it's awesome but now i have a problem when i want to restart the hackintosh, it's reboot but before going to clover, the display becomes twinkling.

Before patching, restart worked perfectly but i have a problem with the native resolution.

http://i.imgur.com/i70fdBL.jpg

My config :
I have a Dell XPS 2720 with : 
- native resolution : 2560x1440 (ok with the patch)
- i7 4770S
- 16Go RAM
- m2sata 128Go (UEFI Clover v2.4k r4173+ dual boot Win 10 Pro 64Bits + Sierra 10.12.6)
- Intel HD 4600 integrated + Nvidia GT 755M

I patched with CoreDisplay + Nvidia but for Nvidia i have this message : 
imac-de-admin:~ admin$ ~/Downloads/NVIDIA-patcher.command 
SIP looks to be disabled, all good!
Keep SIP disabled, if enabled again SIP prevent the Nvidia Driver from loading

Unknown version of the Nvidia driver found..
---- BEGINNING MD5 HASH SUMS ---- version: 10.12.6 build:16G29

     NVDAGK100Hal: 59ea570b00dd9018080220ccb1d3e9b8
 NVDAGK100Hal.bak: NO FILE (this is okay)

---- ENDING MD5 HASH SUMS -------

Please, what can i do to solve this problem ?

I don't want to unptach because the resolution is max 1920x1080 even with SwitchResx, the resolution force to 2560x1440 but everything was blur and the resolution was not enable on display monitor or cards video setting

Thank you very much for your answer

![20170830_224026](https://user-images.githubusercontent.com/31490324/29895090-19bdb970-8dd8-11e7-95b1-2c6a60599dc5.jpg)

**Comment by Floris497** [comment]
Hm.. I have absolutely no idea why this would be happening.. what resolution does the OSD say it is receiving?

**Comment by hyperbroly** [comment]
natively 2560x1440
I don't input the display with any hdmi or dvi Displayport because The DELL XPS 2720 is an AIO (ALL IN ONE) like the Imac 14,2 (smbios that i have put on my hackintosh).
When i turn on, everything is ok, i have clover with Sierra & Windows, Sierra start very well.
If i shutdown, everything is ok, i can turn on again, but when i want to restart only with sierra, this problem happens.

But Floris, i really want to thank you because before patching my hackintosh, i can't upgrade the resolution (still 1920x1080) even switchResx which i had a false blur 2560x1440

**Comment by Floris497** [comment]
Ah restart and sleep sometimes trigger this behavior..it now comes to my mind that another person had this too.. I've had a genuine mac mini do this also.. idk what to do about it.. unfortunately :/

**Comment by hyperbroly** [comment]
Yes my f****g hackintosh had hibernate once !!!
Do you think if i'm unpatch, shut down, turn on and patch again.
After shut down, and try it again.
This solution can fix this problem ?

I really don't understand why only for restarting i have this problem...

**Comment by Floris497** [comment]
Well I can only guess why, when you restart the power to everything isn't completely cut so when reinitializing stuff something goes wrong or something.. 😐 no idea

**Comment by hyperbroly** [comment]
is it possible to log the restart in a file to see what's going on with OSD ?
If yes, how ?
Thank you

**Comment by Floris497** [comment]
No, I didn't know that you had an internal display, those do not have an OSD (On Screen Display) the thing were you can acces some of the settings in the display.
